tests/serialize/t10-ref.ttl

```diff
@@ -1,47 +1,52 @@
 @prefix dc: <http://purl.org/dc/elements/1.1/>.
+@prefix sch: <http://www.w3.org/ns/pim/schedule#>.
 @prefix c: <https://www.w3.org/People/Berners-Lee/card#>.
 @prefix XML: <http://www.w3.org/2001/XMLSchema#>.
-@prefix sch: <http://www.w3.org/ns/pim/schedule#>.
 @prefix sp: <http://www.w3.org/ns/pim/space#>.
 @prefix det: <http://linkeddata.github.io/app-schedule/details.ttl#>.
 @prefix rd: <http://www.w3.org/2000/01/rdf-schema#>.
 @prefix ic: <http://www.w3.org/2002/12/cal/ical#>.
 @prefix n0: <http://xmlns.com/foaf/0.1/>.
 
-<#event>
-    dc:author    c:i;
-    dc:created    "2015-09-15T11:20:05Z"^^XML:dateTime;
-    a    sch:SchedulableEvent;
-    sp:inspiration    det:event1.
-<#event1>
-    dc:author    <#id1442316021316>;
-    dc:title    "TAG 2016 Q1 Meeting ";
-    rd:comment    "Three day meeting, starting date:";
-    ic:location    "Melbourne Australia";
-    sch:availabilityOptions
-       sch:YesNoMaybe;
+:event
```

Comment:
If this is not an official spec, then should we be serializing all of our graphs in this way?